### ISSUE TITLE
groupCall recent_speakers

### ISSUE THREAD

**Issue by zevlg** [tdlib/td#1458]
With TDLib 1.7.3 when someone joins empty groupCall, I receive `updateGroupCall` where "recent_speakers" have 2 members: chat itself, where voice chat is initiated and user who actually joined the voice chat.  After user starts speaking, another `updateGroupCall` event is received, where chat itself is speaking, such as:
```
[ 3][t 4][1616436788.348529100][Td.cpp:4508][#1][!Td][&td_requests]	Sending update: updateGroupCall {
  group_call = groupCall {
    id = 2
    title = "Привет 4"
    is_active = true
    is_joined = false
    need_rejoin = false
    can_unmute_self = false
    can_be_managed = false
    participant_count = 1
    loaded_all_participants = false
    recent_speakers = vector[2] {
      groupCallRecentSpeaker {
        speaker = messageSenderChat {
          chat_id = -1001452936586
        }
        is_speaking = true
      }
      groupCallRecentSpeaker {
        speaker = messageSenderUser {
          user_id = 781215372
        }
        is_speaking = false
      }
    }
    mute_new_participants = true
    can_change_mute_new_participants = false
    record_duration = 0
    duration = 0
  }
}
```

When user stops speaking, another event is received:
```
[ 3][t 4][1616436794.356028318][Td.cpp:4508][#1][!Td][&td_requests]	Sending update: updateGroupCall {
  group_call = groupCall {
    id = 2
    title = "Привет 4"
    is_active = true
    is_joined = false
    need_rejoin = false
    can_unmute_self = false
    can_be_managed = false
    participant_count = 1
    loaded_all_participants = false
    recent_speakers = vector[2] {
      groupCallRecentSpeaker {
        speaker = messageSenderChat {
          chat_id = -1001452936586
        }
        is_speaking = false
      }
      groupCallRecentSpeaker {
        speaker = messageSenderUser {
          user_id = 781215372
        }
        is_speaking = false
      }
    }
    mute_new_participants = true
    can_change_mute_new_participants = false
    record_duration = 0
    duration = 0
  }
}
```

What does that mean?  I want to outline user, while he is speaking, but looks like there not enough info in the events to do so.
What that `messageSenderChat` means being in the "recent_speakers" list?

Thanks

**Comment by levlam** [comment]
Do you have TDLib's log with verbosity level 4?

**Comment by levlam** [comment]
Thanks. Fixed in 9936cdc320c07b2e24fb9b595870e049b3ea4322.

**Comment by zevlg** [comment]
Works! Thank you
